src/test/mockapi/tst101.json

```diff
@@ -0,0 +1,75 @@
+{
+  "TEST": {
+    "courseId": "00001",
+    "org": "TST",
+    "orgName": "Test",
+    "courseTitle": "Test Json",
+    "code": "TST101",
+    "section": "Y",
```

Comment:
i'm currently augmenting the course class to have exclusions as an attribute, do you think you can add an exclusion key to the mock JSON so that I can use it for testing?

Reply:
can do

Reply:
ok i added some very basic exclusions. if you want more for your testing, you can just make new mock api files with the template